pkg/stanza/fileconsumer/matcher/matcher.go

```diff
@@ -94,7 +109,6 @@ func New(c Criteria) (*Matcher, error) {
 		}
 	}
 
-	var filterOpts []filter.Option
```

Comment:
It's ok to leave this out but let's just append directly to `m.filterOpts`

Reply:
Thanks for catching; fixed in a18957bf6f.